components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -564,6 +576,50 @@ void RRTMGPRadiation::run_impl (const double dt) {
     shr_orb_decl_c2f(calday, eccen, mvelpp, lambm0,
                      obliqr, &delta, &eccf);
 
+    // Precompute VMR for all gases, on all cols, before starting the chunks loop
```

Comment:
I think at some point we should maybe think about moving this out of radiation, or at least moving out of run_impl to clean this up. I think I could do this when I tackle generalizing the diagnostic rad calls (clear sky, clean sky, etc).